database/migrations/2024_03_09_004053_update_unique_template_id_default_value_in_screen_templates.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('screen_templates', function (Blueprint $table) {
+            $table->string('unique_template_id')->nullable()->change();
```

Comment:
As an observation, I think that this value is not being used yet. It appears empty in my template, If not needed we can consider removing it

Reply:
@eiresendez This value will be used by the Default Templates created by the PLG team